types/v56/types.go

```diff
@@ -2626,13 +2682,15 @@ type VmQuestionAnswer struct {
 // Reference: vCloud API 27.0 - DiskType
 // https://code.vmware.com/apis/287/vcloud#/doc/doc/types/QueryResultDiskRecordType.html
 type DiskRecordType struct {
-	Xmlns              string  `xml:"xmlns,attr,omitempty"`
-	HREF               string  `xml:"href,attr,omitempty"`
-	Id                 string  `xml:"id,attr,omitempty"`
-	Type               string  `xml:"type,attr,omitempty"`
-	Name               string  `xml:"name,attr,omitempty"`
-	Vdc                string  `xml:"vdc,attr,omitempty"`
-	SizeB              int64   `xml:"sizeB,attr,omitempty"`
+	Xmlns string `xml:"xmlns,attr,omitempty"`
+	HREF  string `xml:"href,attr,omitempty"`
+	Id    string `xml:"id,attr,omitempty"`
+	Type  string `xml:"type,attr,omitempty"`
+	Name  string `xml:"name,attr,omitempty"`
+	Vdc   string `xml:"vdc,attr,omitempty"`
+	// SizeB is not available in API V33.0
```

Comment:
I have commented this out on purpose so that one can immediately see a breaking change. Other option could be to leave it and let API return the error.